### PR TITLE
update extension docs

### PR DESCRIPTION
**Pull request by erwinmombay**
Closes https://github.com/ampproject/amphtml/issues/3714

### CODE REVIEW

**Comment by ericlindley-g** [comment]
LGTM (@erwinmombay — holding off on removing "needs" and adding "LGTM" to labels until @jmadler gets a chance to look, but let me know if I should just go ahead)

File: extensions/README.md [hunk 1]
```diff
@@ -27,9 +28,11 @@ Current list of extended components:
 | [`amp-carousel`](amp-carousel/amp-carousel.md) | Displays multiple similar pieces of content along a horizontal axis. |
 | [`amp-dailymotion`](amp-dailymotion/amp-dailymotion.md) | Displays a [Dailymotion](https://www.dailymotion.com) video. |
 | [`amp-dynamic-css-classes`](amp-dynamic-css-classes/amp-dynamic-css-classes.md) | Adds several dynamic CSS class names onto the HTML element. |
+| [`amp-embed`](../builtins/amp-embed.md) | Embed elements into the AMP page. Can be used instead of `amp-ad` when semantically more accurate.  |
```
Comment (jmadler) [inline extensions/README.md]: This wording could be more concise to clarify that amp-embed is only useful as an alias of amp-ad

Comment (erwinmombay) [inline extensions/README.md]: any suggestions? this is the current wording on the embed docs

**Comment by erwinmombay** [comment]
superceded by latest extension list update

**Pull request closed without merging**